src/pages/GivePage/index.tsx

```diff
@@ -357,15 +357,31 @@ const GivePage = () => {
   }, [startingTeammates, allUsers]);
 
   useEffect(() => {
-    if (pendingGiftsFrom) {
-      setGifts(
-        pendingGiftsFrom.reduce<typeof gifts>((map, g) => {
-          map[g.recipient_id] = g;
-          return map;
-        }, {})
-      );
+    if (!pendingGiftsFrom) return;
+
+    // set default donation to Coordinape to 1% (rounded down)
+    if (pendingGiftsFrom.length === 0) {
+      const grantee = allUsers?.find(isUserCoordinape);
+      if (grantee)
+        setGifts({
+          [grantee.id]: {
+            note: '',
+            recipient_id: grantee.id,
+            tokens: Math.floor(myUser.starting_tokens / 100),
+          },
+        });
+
+      return;
     }
-  }, [pendingGiftsFrom]);
+
+    // load previously-saved allocations
+    setGifts(
+      pendingGiftsFrom.reduce<typeof gifts>((map, g) => {
```

Comment:
can spell out `g` as gift

Reply:
thanks for the suggestion. in cases like this, where we've already spelled out the name of the thing nearby and the function is small, i prefer brevity